apps/functions/applications-migration/common/migrators/nsip-project-migration.js

```diff
@@ -59,22 +59,13 @@ const getNsipProjects = async (log, caseReference) => {
 	log.info(`Retrieved projects ${JSON.stringify(projects)}`);
 
 	return projects.map((project) => ({
-		dateIAPIDue: null,
-		jRPeriodEndDate: null,
-		operationsLeadId: null,
-		operationsManagerId: null,
-		caseManagerId: null,
-		nsipOfficerIds: [],
-		nsipAdministrationOfficerIds: [],
-		leadInspectorId: null,
-		inspectorIds: [],
-		environmentalServicesOfficerId: null,
-		legalOfficerId: null,
-		applicantId: null,
-		migrationStatus: null,
-		dateOfReOpenRelevantRepresentationStart: null,
-		dateOfReOpenRelevantRepresentationClose: null,
 		...project,
-		regions: project.region ? [project.region] : []
+		nsipOfficerIds: valueToArray(project.nsipOfficerIds),
+		nsipAdministrationOfficerIds: valueToArray(project.nsipAdministrationOfficerIds),
+		inspectorIds: valueToArray(project.inspectorIds),
+		migrationStatus: project.migrationStatus ? Boolean(project.migrationStatus) : null,
```

Comment:
This is going to only return `true` or `null`, but `false` is a valid value. We probably want to default to `false`, because `null` will just be for those projects that are created new in the back office.

Reply:
Thanks, that actually simplifies the logic a bit. If we get null/undefined, then `Boolean` will return `false`.